src/components/atoms/Modal.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
```

Comment:
There is already a modal component in `src/components/organisms/modal/Modal.tsx`